app/src/pages/organization/projectTeamPage/membersListTable/membersListTable.jsx

```diff
@@ -0,0 +1,189 @@
+/*!
+ * Copyright 2024 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import classNames from 'classnames/bind';
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+import { useMemo } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { activeProjectKeySelector, userRolesSelector } from 'controllers/user';
+import { ADMINISTRATOR } from 'common/constants/accountRoles';
+import { EDITOR, MANAGER, VIEWER } from 'common/constants/projectRoles';
+import { AbsRelTime } from 'components/main/absRelTime';
+import { MeatballMenuIcon, Popover, Table } from '@reportportal/ui-kit';
+import { UserAvatar } from 'pages/inside/common/userAvatar';
+import { urlOrganizationAndProjectSelector } from 'controllers/pages';
+import { SORTING_ASC, withSortingURL } from 'controllers/sorting';
+import { DEFAULT_SORT_COLUMN } from 'controllers/administrate/allUsers';
+import { fetchMembersAction } from 'controllers/members';
+import { messages } from './messages';
+import styles from './membersListTable.scss';
+
+const cx = classNames.bind(styles);
+
+export const MembersListTableWrapped = ({ members, onChangeSorting, sortingDirection }) => {
+  const { formatMessage } = useIntl();
+  const dispatch = useDispatch();
+  const activeProjectKey = useSelector(activeProjectKeySelector);
+  const { organizationSlug, projectSlug } = useSelector(urlOrganizationAndProjectSelector);
+  const { userRole: role, organizationRole, projectRole } = useSelector(userRolesSelector);
+  const hasPermission = role === ADMINISTRATOR || organizationRole === MANAGER;
```

Comment:
Please use the permission defined in common permissions, let's try to avoid deciding which role can act in particular case in components.

Reply:
Done.